src/Files.App/Helpers/DynamicDialogFactory.cs

```diff
@@ -134,5 +134,85 @@ public static DynamicDialog GetFor_FileInUseDialog(List<Shared.Win32Process> loc
 			});
 			return dialog;
 		}
+
+		public static DynamicDialog GetFor_CredentialEntryDialog(string path)
+		{
+			string[] userAndPass = new string[3];
+			DynamicDialog? dialog = null;
+
+			TextBox inputUsername = new()
+			{
+				Height = 35d,
```

Comment:
Does the height need to be hardcoded?

Reply:
Sounds good, I've removed the heights I added and from the rename dialog as well.